echopype/echodata/echodata.py

```diff
@@ -143,35 +143,48 @@ def compute_range(
 
         This method only applies to `sonar_model`s of `"AZFP"`, `"EK60"`, and `"EK80"`.
         If the `sonar_model` is not `"AZFP"`, `"EK60"`, or `"EK80"`, an error is raised.
-
         If the `sonar_model` is `"AZFP"`, the returned range's data will be duplicated
         for all `ping_time`s.
 
         Parameters
         ----------
         env_params: dict
             This dictionary should contain either:
+
             - `"sound_speed"`: `float`
-            - `"temperature"`, `"salinity"`, and `"pressure"`: `float`s,
+            - `"temperature"`, `"salinity"`, and `"pressure"`: `float`s
+
             in which case the sound speed will be calculated.
             If the `sonar_model` is `"EK60"` or `"EK80"`, and
             `EchoData.environment.sound_speed_indicative` exists, then this parameter
             does not need to be specified.
+
         azfp_cal_type : {"Sv", "Sp"}, optional
+
             - `"Sv"` for calculating volume backscattering strength
             - `"Sp"` for calculating point backscattering strength.
+
             This parameter is only used if `sonar_model` is `"AZFP"`,
             and in that case it must be specified.
+
```

Comment:
Same comment as in `compute_Sv` about the ambiguous statement. What does "it" in "it must be specified" refer to? `sonar_model`? If that's the case, can't `sonar_model` be read directly from the echodata metadata?

Reply:
docstrings were overhauled